lib/init/grass7.html

```diff
@@ -110,6 +110,23 @@ <h2>DESCRIPTION</h2>
 these checks fail then <em>grass79</em> will automatically switch back
 to the text user interface mode.
 
+<h2>FLAGS</h2>
+
+The flag <b>--config option</b> prints GRASS GIS configuration and
+version parameters, with the options:
+
+<ul>
+<li><b>arch</b>: system architecture (e.g., <tt>x86_64-pc-linux-gnu</tt>)</li>
+<li><b>build</b>: (e.g., <tt>./configure --with-cxx --enable-largefile --with-proj [...]</tt>)</li>
+<li><b>compiler</b>: (e.g., <tt>gcc</tt>)</li>
+<li><b>date</b>: (e.g., <tt>Tue Mar 31 20:34:57 2020 +0200</tt>)</li>
+<li><b>path</b>: (e.g., <tt>/usr/lib64/grass78</tt>)</li>
+<li><b>revision</b>: (e.g., <tt>745ee7ec9</tt>)</li>
+<li><b>svn_revision</b>: (e.g., <tt>062bffc8</tt>)</li>
+<li><b>version</b>: (e.g., <tt>7.9.dev</tt>)</li>
```

Comment:
Wait, I actually assumed in my previous comment that the answer is that this is WIP. I do get the same info from running the command, but I don't think people will guess what `svn_revision` refers to. I'm not sure what it is now without checking the code.

Reply:
We may mark `svn_revision` here as deprecated.

It refers to `VERSIONNUMBER`:
https://github.com/OSGeo/grass/blob/b74dfd2f5045864dab6037d54a1d610b278417d9/lib/init/grass.py#L2023

Reply:
@landam suggested `libgis_revision` (but only in GRASS 8), see https://github.com/OSGeo/grass/pull/405#issuecomment-616767468

Reply:
VERSIONNUMBER which actually refers to ...? Whole include directory, i.e., the whole C/C++ API/ABI I guess. The other candidates I can think of are gis.h, libgrass_gis.so source code, or GRASS version.

We can add the new name now. We just have to keep the old one as @landam suggests.

Reply:
@wenzeslaus I think adding a new config option `libgis_revision` should be part of a different PR since Python code is involved.